modules/understanding-upgrade-channels.adoc

```diff
@@ -7,159 +7,145 @@
 // * updating/updating-disconnected-cluster.adoc
 
 [id="understanding-upgrade-channels_{context}"]
-= {product-title} Upgrade Channels and Releases
+= Understanding {product-title} upgrade channels
 
 In {product-title} 4.1, Red Hat introduced the concept of channels for
 recommending the appropriate release versions for cluster upgrade. By controlling
-the pace of upgrades, these upgrade channels allow users to choose an upgrade
+the pace of upgrades, these upgrade channels allow you to choose an upgrade
 strategy. Upgrade channels are tied to a minor version of
 {product-title}. For instance, {product-title} {product-version}
-upgrade channels will never include an upgrade to a 4.5 release. This ensures
-administrators make an explicit decision to upgrade to the next minor version of
-{product-title}. Upgrade channels only control release selection and have
-no impact on the version of the cluster you install; the `openshift-install`
-binary for a given patch level of {product-title} always installs that patch level.
+upgrade channels will never include an upgrade to a 4.5 release. This strategy ensures that
+administrators explicitly decide to upgrade to the next minor version of
+{product-title}. Upgrade channels control only release selection and do not impact the version of the cluster that you install; the `openshift-install`
+binary file for a specific version of {product-title} always installs that version.
 
-{product-title} {product-version} has the following upgrade channels to choose from:
+{product-title} {product-version} offers the following upgrade channels:
 
-* candidate-{product-version}
-* fast-{product-version}
-* stable-{product-version}
+* `candidate-{product-version}`
+* `fast-{product-version}`
+* `stable-{product-version}`
 
 [discrete]
-== candidate-{product-version} Channel
+== candidate-{product-version} channel
 
-The candidate-{product-version} channel will contain candidate builds for a z-stream
+The `candidate-{product-version}` channel contains candidate builds for a z-stream
 ({product-version}.z) release.
-Release candidates contain all the features of the product but they are not supported and
-should only be used to test feature acceptance and assist in qualifying the next version
+Release candidates contain all the features of the product but are not supported. Use release candidate versions to test feature acceptance and assist in qualifying the next version
 of {product-title}.
-A release candidate is any build (e.g. 4.3.0-rc.3, 4.3.0) that is available in the candidate
+A release candidate is any build, such as `4.3.0-rc.3` that is available in the candidate
```

Comment:
We do want the other build, 4.3.0, mentioned as well to stress that "release candidate" does not only refer to a build  with the '-rc' extension.

Reply:
> If that's what you're trying to emphasize, I think it would be clearer to say...

I like the wording you propose here, but you seem to have marked this thread resolved without updating the pull request to include it?